packages/react-import-remote/src/test/ImportRemote.test.tsx

```diff
@@ -163,16 +166,12 @@ describe('<ImportRemote />', () => {
       );
       expect(load).not.toHaveBeenCalled();
 
-      expect(importRemote.find(IntersectionObserver)).toHaveProp(
-        'threshold',
-        0,
-      );
+      expect(importRemote.find(IntersectionObserver)).toHaveReactProps({
+        threshold: 0,
+      });
 
-      await trigger(
-        importRemote.find(IntersectionObserver),
-        'onIntersectionChange',
-        {isIntersecting: true},
-      );
+      // perform something that gets us into view
```

Comment:
Not sure how to do the "perform" part of this test now.

Reply:
Not sure you can other than just mocking out the inner hook, or mocking whatever it uses to decide to fire an event.